app/controllers/movies_controller.rb

```diff
@@ -1,8 +1,23 @@
 class MoviesController < ApplicationController
-
-  
   def index
-      @movies = Movie.page(params[:page]).per(18).all
+    if params[:genre].nil? 
+      @movies = Movie.page(params[:page]).per(18).where(genre: ["Ruby on Rails", "Git","Basic","Ruby"])
+    elsif params[:genre] == "Live"
+      @movies = Movie.page(params[:page]).per(18).where(genre: "Live")
+    elsif params[:genre] == "Talk"
+      @movies = Movie.page(params[:page]).per(18).where(genre: "Talk")
+    elsif params[:genre] == "Movie"
+      @movies = Movie.page(params[:page]).per(18).where(genre: "Movie")
+    elsif params[:genre] == "Php"
+      @movies = Movie.page(params[:page]).per(18).where(genre: "Php")
+    elsif params[:genre] == "Writing"
+      @movies = Movie.page(params[:page]).per(18).where(genre: "Writing")
+    elsif params[:genre] == "Marketing"
+      @movies = Movie.page(params[:page]).per(18).where(genre: "Marketing")
+    elsif params[:genre] == "Money"
+      @movies = Movie.page(params[:page]).per(18).where(genre: "Money")
+    end
+      # @movies = Movie.page(params[:page]).per(18).all
```

Comment:
`where` を先に書かないと動画数が多い時に挙動がおかしくなる可能性があると思います。
また，各ジャンルの表示はまとめられますね。

```suggestion
    if params[:genre].nil? 
      @movies = Movie.where(genre: ["Basic", "Git", "Ruby", "Ruby on Rails"]).page(params[:page]).per(18)
    else
      @movies = Movie.where(genre: params[:gemre]).page(params[:page]).per(18)
    end
```